cve_bin_tool/package_list_parser.py

```diff
@@ -167,10 +167,10 @@ def add_vendor(self, vendor_package_pairs):
             for package_name in self.package_names_without_vendor:
                 if vendor_package_pair["product"] == package_name["name"].replace(
                     "*", ""
-                ):
-                    package_name["vendor"] = vendor_package_pair["vendor"] + "*"
-                    self.package_names_with_vendor.append(package_name)
-                    self.package_names_without_vendor.remove(package_name)
+                ):  
```

Comment:
```suggestion
                ):
```

Needs trailing whitespace removed for flake8 linter.